### PR TITLE
IMa2p built using foss 2016a

### PR DESCRIPTION
**Pull request by FokkeDijkstra**
IMa2p built using foss 2016a

### CODE REVIEW

File: easybuild/easyconfigs/i/IMa2p/IMa2p-20151123-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/arunsethuraman/ima2p/archive/']
+sources = ['%s.zip' % commit]
+
+# If CXX is already set the compiler will not be changed to mpic++ by 
+# the configure. We therefore set it to mpic++.
+preconfigopts = 'chmod +x configure && export CXX=mpic++ && '
```
Comment (boegel) [inline easybuild/easyconfigs/i/IMa2p/IMa2p-20151123-foss-2016a.eb]: another option is to use `toolchainopts = {'usempi': True}` which will make EasyBuild set `$CXX` to the MPI C++ compiler wrapper (+ more, like defining `$OMPI_CXX`)

**Comment by FokkeDijkstra** [comment]
boegel, thanks for the comment. I should have realized this earlier.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2430.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2f38ca88e909f3e8a3986e9514fcbfb0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2010.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b8bf6c33364ca0897672fc3e48d8fb15 for a full test report.

**Comment by boegel** [comment]
lgtm

**Pull request merged**